app/src/main/java/com/ciandt/dragonfly/example/features/projectselection/ProjectSelectionInteractor.kt

```diff
@@ -0,0 +1,41 @@
+package com.ciandt.dragonfly.example.features.projectselection
+
+import android.content.Context
+import android.os.AsyncTask
+import com.ciandt.dragonfly.example.data.ProjectRepository
+import com.ciandt.dragonfly.example.models.Project
+
+class ProjectSelectionInteractor(val context: Context) : ProjectSelectionContract.Interactor {
+
+    override fun loadProjects(onSuccess: (List<Project>) -> Unit, onFailure: (Exception) -> Unit) {
+        LoadProjectsTask(context, onSuccess, onFailure).execute()
```

Comment:
Maybe a extension with executeParallel() = this.executeOnExecuter(AsyncTask.THREAD_POOL_EXECUTOR)